ui/src/app/shared/services/requests.ts

```diff
@@ -27,7 +27,7 @@ let baseHRef = '/';
 const onError = new BehaviorSubject<agent.ResponseError>(null);
 
 function toAbsURL(val: string): string {
-    return path.join(baseHRef, val);
+    return `${_.trimEnd(baseHRef, '/')}${val}`;
```

Comment:
lodash seems not used

Reply:
`import _ from 'lodash';`

`_` is the common alias for the Lodash library

Reply:
Ah, I see